oonib/api.py

```diff
@@ -1,29 +1,43 @@
 from cyclone import web
 
-from oonib.main.api import mainAPI
-from oonib.deck.api import deckAPI
-from oonib.report.api import reportAPI
-from oonib.input.api import inputAPI
-from oonib.policy.api import policyAPI
 from oonib.bouncer.api import bouncerAPI
```

Comment:
For symmetry with moving APIs in class scope you would probably want to move also this statement